build.xml

```diff
@@ -662,32 +662,6 @@
         </if>
     </target>
 
-    <target name="timezones">
```

Comment:
although this will work perfectly in case of creating build
but for development environment we are using prod-deploy target which will be broken with this because after deploying the war we will not be copying the timezone related files